frontend/src/state-providers/fonts.ts

```diff
@@ -46,11 +46,11 @@ export function createFontsState(editor: Editor) {
 
 		const response = await fetch(fontFileUrl);
 		const responseBuffer = await response.arrayBuffer();
-		editor.instance.on_font_load(fontFileUrl, new Uint8Array(responseBuffer), true);
+		editor.instance.on_default_font_load(fontFileUrl, new Uint8Array(responseBuffer));
 	});
 	editor.subscriptions.subscribeJsMessage(TriggerFontLoad, async (triggerFontLoad) => {
```

Comment:
Can we rename the `TriggerFontLoad` message to `TriggerFontLoadUrl`, then rename `TriggerFontLoadDefault` to `TriggerFontLoad` and make it pass along a `font_family` and `font_style`? Then the backend can request the default font by family/style instead of URL. Meaning we'd move these into the backend:
```ts
const DEFAULT_FONT = "Merriweather";
const DEFAULT_FONT_STYLE = "Normal (400)";
```